man/avocado.rst

```diff
@@ -13,13 +13,13 @@ DESCRIPTION
 ===========
 
 Avocado is a modern test framework that is built on the experience
-accumulated with `autotest` (`http://autotest.github.io`).
+accumulated with `autotest` (https://autotest.github.io).
```

Comment:
You are removing the ` here, but keeping on the following changes. Any reason for that?

Reply:
The `  in the link do nothing and are ignored. You can compare in the current man page with the link to  http://avocado-framework.github.io/ shown a couple of lines below. They're both rendered the same way. It'd be the same if you were rendering as a HTML instead, for example, see current https://github.com/avocado-framework/avocado/blob/master/man/avocado.rst

Reply:
I'm ok with removing this, I just would like to understand why only removed here and left others that you touched. So I suggest to stick with one way instead of having some with and others without.

Reply:
@beraldoleal I just noticed the others now. I'll update and remove all the backticks.